doc/examples/filters/plot_denoise.py

```diff
@@ -24,13 +24,24 @@
 A bilateral filter is an edge-preserving and noise reducing filter. It averages
 pixels based on their spatial closeness and radiometric similarity.
 
+Wavelet denoising filter
+------------------------
+
+A wavelet denoising filter relies on the wavelet representation of the image.
+The noise is represented by small values in the wavelet domain which are set to
+0.
+
+In color images, wavelet denoising is typically done in the YCbCr color space
```

Comment:
Is it possible to add a link to https://en.wikipedia.org/wiki/YCbCr in this sentence please?